terraform/custom_domains/environment_domains/terraform.tf

```diff
@@ -1,10 +1,10 @@
 terraform {
 
-  required_version = "= 1.3.5"
+  required_version = "= 1.5.4"
```

Comment:
is there a way we can lock this to only the latest minor version instead?

Reply:
From experience we found it's safer be as explicit as possible so everyone uses the same exact version and updates the same state file accordingly. And version updates are done explicitly via PR.
I've added an entry for [asdf](https://technical-guidance.education.gov.uk/infrastructure/dev-tools/#asdf) in this PR to help moving between version seemlessly.